tsl/src/backup.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+#include <postgres.h>
+#include <funcapi.h>
+#include <utils/builtins.h>
+#include <utils/pg_lsn.h>
+#include <access/xlog_internal.h>
+#include <access/xlog.h>
+#include <access/xact.h>
+#include <catalog/pg_foreign_server.h>
+#include <storage/lmgr.h>
+#include <miscadmin.h>
+
+#include "errors.h"
+#include "guc.h"
+#include "catalog.h"
+#include "debug_wait.h"
+#include "dist_util.h"
+#include "remote/dist_commands.h"
+#include "backup.h"
+
+enum
+{
+	Anum_restore_point_node_name = 1,
+	Anum_restore_point_lsn,
+	_Anum_restore_point_max
+};
+
+static Datum
+create_restore_point_datum(TupleDesc tupdesc, const char *node_name, XLogRecPtr lsn)
+{
+	Datum values[_Anum_restore_point_max];
+	bool nulls[_Anum_restore_point_max] = { false };
+	HeapTuple tuple;
+
+	tupdesc = BlessTupleDesc(tupdesc);
+	values[AttrNumberGetAttrOffset(Anum_restore_point_node_name)] = CStringGetDatum(node_name);
```

Comment:
The idea of this line was to set NULL in value as well without adding it under if, since CStringGetDatum does not do any transformations. I guess I can make it more clear as you suggest